tests/pages/home.py

```diff
@@ -2,9 +2,54 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+from selenium.webdriver.support.ui import WebDriverWait as Wait
+
 from .base import BasePage
+from .page import PageRegion
 
 
 class HomePage(BasePage):
 
-    pass
+    _promo_grid_locator = (By.CSS_SELECTOR, '.promo-grid.reveal')
+    _promo_tile_locator = (By.CSS_SELECTOR, '.promo-grid > .item')
+    _promo_tweet_locator = (By.ID, 'twt-body')
+    _download_button_locator = (By.ID, 'download-button-desktop-release')
+
+    @property
+    def is_promo_grid_displayed(self):
+        Wait(self.selenium, self.timeout).until(
+            expected.visibility_of_element_located(self._promo_grid_locator))
+        return self.selenium.find_element(*self._promo_grid_locator).is_displayed()
+
+    @property
+    def expected_promos(self):
+        return len(self.selenium.find_elements(*self._promo_tile_locator))
```

Comment:
Just a caution, but you're returning all promos here regardless of whether they're displayed. I know you've mentioned that you're concerned about checking for visibility of certain promos. I might suggest at least renaming the property to `number_of_promos_present` for a little bit of extra clarity.